go/internal/domain/repository/timeline_items.go

```diff
@@ -5,7 +5,7 @@ import "x-clone-backend/internal/domain/entity"
 type TimelineItemsRepository interface {
 	SpecificUserPosts(userID string) ([]*entity.TimelineItem, error)
 	UserAndFolloweePosts(userID string) ([]*entity.TimelineItem, error)
-	CreatePost(userID, text string) (entity.TimelineItem, error)
+	CreatePost(userID, text string, usersID []string) (entity.TimelineItem, error)
```

Comment:
Please double-check before marking this as resolved.
```suggestion
	CreatePost(userID, text string, userIDs []string) (entity.TimelineItem, error)
```

Reply:
I will use the folder search in VSCode; therefore, it will never happen again.